docs/pages/chat-apps/list-stream-sync/sync-and-syncall.mdx

```diff
@@ -1,7 +1,9 @@
 # Sync conversations and messages
 
 :::tip[Note]
-Syncing does not refetch existing conversations and messages. It also does not fetch messages for group chats you are no longer a part of.
+
```

Comment:
We've had this note up top for a while -- I am wondering if we should remove it. 

The first sentence is untrue -- or stated incorrectly? I think the info is better handled in the syncAll section specifically. And do we need to state the second sentence?

Reply:
I personally find it confusing. When I really think about it, I can understand what "does not refetch existing conversations and messages" means, but it's not intuitive.

I think we can ditch.